### PR TITLE
Debian packaging cleanups

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: tools/debian/rules [hunk 1]
```diff
@@ -38,9 +38,7 @@ override_dh_install:
 	rm debian/tmp/etc/motd.d/cockpit debian/tmp/etc/issue.d/cockpit.issue
 
 	# unpackaged modules
-	for m in kdump selinux; do rm -r debian/tmp/usr/share/cockpit/$$m; done
-	rm debian/tmp/usr/share/metainfo/org.cockpit-project.cockpit-kdump.metainfo.xml
-	rm debian/tmp/usr/share/metainfo/org.cockpit-project.cockpit-selinux.metainfo.xml
+	cd debian/tmp/usr/share; for m in kdump selinux; do rm -r cockpit/$$m metainfo/org.cockpit-project.cockpit-$$m.metainfo.xml; done
```
Comment (allisonkarlitskaya) [inline tools/debian/rules:41]: ugh.  Okay.  This is a Makefile.  This is fine.  But it sure looks awful :rofl:

Comment (allisonkarlitskaya) [reply tools/debian/rules:41]: Honestly, can you just take this in the other direction and delete all 4 files explicitly with full path names?

Comment (martinpitt) [reply tools/debian/rules]: Sure! done.

**Review by allisonkarlitskaya — APPROVED** [review]
looks good, thanks

**Pull request merged**